src/services/DiscourseService.ts

```diff
@@ -0,0 +1,62 @@
+import logger from 'decentraland-gatsby/dist/entities/Development/logger'
+import { Avatar } from 'decentraland-gatsby/dist/utils/api/Catalyst'
+
+import { Discourse, DiscoursePost } from '../clients/Discourse'
+import { DISCOURSE_AUTH, DISCOURSE_CATEGORY } from '../entities/Discourse/utils'
+import { inBackground } from '../entities/Proposal/routes'
+import * as templates from '../entities/Proposal/templates'
+import { forumUrl, proposalUrl } from '../entities/Proposal/utils'
+
+import { ProposalInCreation } from './ProposalService'
+import { SnapshotService } from './SnapshotService'
+
+export class DiscourseService {
+  static async createProposal(
+    data: ProposalInCreation,
+    proposalId: string,
+    profile: Avatar | null,
+    snapshot_url: string,
+    snapshotId: string
+  ) {
+    let discourseProposal: DiscoursePost
```

Comment:
You can move this as a const inside the try with the logger call and the return.